### ISSUE TITLE
Commandbar: Screen reader is not announcing the expanded/collapsed state of Overflow command button

### ISSUE THREAD

**Issue by raghavendra-kodimala** [OfficeDev/office-ui-fabric-react#8297]
### Environment Information 
 
- __Package version(s)__: (5.122.3) 
- __Browser and OS versions__: (Chrome: 72.0.3626.121 (Official Build) (64-bit), Microsoft Edge: 44.17763.1.0, Microsoft Windows 1809)

#### Actual behavior:
 
Screen reader (Narrator, JAWS, NVDA) is not announcing the expanded/collapsed state of Commandbar overflow button
 
#### Expected behavior:
 
Screen reader should announce the expanded/collapsed state of the overflow command button of commandbar

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? ( No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)

**Comment by KevinTCoughlin** [comment]
Hey @raghavendra-kodimala, I just tried this with Edge + Narrator and the collapsed / expanded state of the overflow button on https://developer.microsoft.com/en-us/fabric#/components/commandbar is announced properly:

```shell
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
```

In the above, I was toggling the overflow button menu via keyboard.

We focus on compatibility with Edge + Narrator. Since Narrator is announcing this properly, perhaps there is an issue that should be opened against NVDA / JAWS.

If you have any guidance on modifications we can make to fix those scenarios, please let us know and we will patch. Thank you!

**Comment by raghavendra-kodimala** [comment]
Hey @KevinTCoughlin , Here are the repro steps that I have followed in the same URL  https://developer.microsoft.com/en-us/fabric#/components/commandbar#Variants

Pre-requisites: Open the URL in MS Edge.
1. Start Narrator
2. Navigate to the "More Commands" button.
3. Press Enter Key to toggle menu without moving to menu item.
Actual Result:
I can still repro the issue. Narrator is not announcing the expanded status by Enter Key press. It's the same case with mouse click.
Expected Result:
Screen reader should read "expanded" when menu opened and should read as "collapsed" when menu dismissed.

**Comment by KevinTCoughlin** [comment]
> Hey @KevinTCoughlin , Here are the repro steps that I have followed in the same URL https://developer.microsoft.com/en-us/fabric#/components/commandbar#Variants
> 
> Pre-requisites: Open the URL in MS Edge.
> 
> 1. Start Narrator
> 2. Navigate to the "More Commands" button.
> 3. Press Enter Key to toggle menu without moving to menu item.
>    **Actual Result:**
>    I can still repro the issue. Narrator is not announcing the expanded status by Enter Key press. It's the same case with mouse click.
>    **Expected Result:**
>    Screen reader should read "expanded" when menu opened and should read as "collapsed" when menu dismissed.
>    Note: On menu opened, focus is moving to menu item. Please try with focus should remain on the button when menu is opened

I'm not hearing the same read out on a different device. Sharing browser specs:

```shell
Microsoft Edge 44.17763.1.0
Microsoft EdgeHTML 18.17763
```

@raghavendra-kodimala can you share your version numbers via Edge -> General Settings -> About this app? I'll cross-reference with the original device used to debug.

**Comment by raghavendra-kodimala** [comment]
@KevinTCoughlin Here is the codepen that I have created. https://codepen.io/raghavendra-kodimala/pen/mopNqQ.
I have the same browser version that you given.
![image](https://user-images.githubusercontent.com/48360110/54336668-b33ea300-4652-11e9-8fec-1454b2592281.png)

**Comment by KevinTCoughlin** [comment]
@raghavendra-kodimala I'm on the same build on my original device.

While I do no longer hear Narrator announce "button expanded", I _do_ see it picked up by Narrator's parser. See:

```
More commands, button expanded,
 Move to..., menu item,
Use left and right arrow keys to navigate between commands, More commands, button collapsed,
Space
More commands, button expanded,
 Move to..., menu item,
```

So I'm unsure what we are doing incorrectly within OUFR if at all or if this is a Narrator bug. I will need to follow-up with that team...

**Comment by raghavendra-kodimala** [comment]
@KevinTCoughlin I have shared with you a OneDrive for business link having the screen recording of what Narrator speaks. Please go through that and let me know if you have queries.

**Comment by khmakoto** [comment]
Hi @raghavendra-kodimala, apparently this is by default in Narrator+Edge.

From the information we got from the Narrator team:

>As soon as you click on the button, focus moves to the first menu item, which does not carry any information about the "expanded" state of the button that triggered it. While Narrator could potentially deal with this better, I think everything works as expected in this scenario, even though it may not provide the best possible experience.

Given that answer I'll be resolving this as `By Design` and `External`. Please let me know if there is anything else I can help with!
